framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -608,6 +642,28 @@ private CheckerMessage(
             this.checker = checker;
         }
 
+        /**
+         * Constructor method for checker message if stack trace needs to be stored.
+         *
+         * @param kind kind of diagnostic, for example, error or warning
+         * @param message error message that needs to be printed
+         * @param source tree element causing the error
+         * @param checker the type-checker in use
+         * @param trace the stack trace when the checker encounters an error
+         */
+        private CheckerMessage(
+                Diagnostic.Kind kind,
+                String message,
+                Tree source,
+                BaseTypeChecker checker,
+                StackTraceElement[] trace) {
```

Comment:
Also nullable.

Reply:
I have made the changes in my latest commit